### DISCUSSION TITLE
[1069690]欺诈领主RogueLords

### DISCUSSION THREAD

**Discussion by haojih0** [pjy612/SteamManifestCache#188] [Share]
appid:1069690 
keys:
> ```
> 1069691;b0b62c3ca8b5fdeae36530e7baa52e37ed21aee3ba813044d12e4458ff5c4240
> ```

[欺诈领主RogueLords.zip](https://github.com/pjy612/SteamManifestCache/files/13778589/RogueLords.zip)

内测码：7EF9DDC35B3032E3
邮箱：1290016861@qq.com

**Comment by pjy612** [comment]
请把密钥贴出来

**Reply by haojih0** [reply]
DecryptionKey：b0b62c3ca8b5fdeae36530e7baa52e37ed21aee3ba813044d12e4458ff5c4240

**Reply by pjy612** [reply]
请编辑自己的原帖

**Reply by haojih0** [reply]
已添加

**Reply by pjy612** [reply]
一点格式不看啊。。。

**Reply by haojih0** [reply]
抱歉抱歉，现在应该行了

**Reply by pjy612** [reply]
？？？你这还是没对上啊
1069690
1096960

**Reply by haojih0** [reply]
(=ＴェＴ=)我傻B了૮ ºﻌºა

**Reply by pjy612** [reply]
查收
